charts/camunda-platform/templates/connectors/templates/deployment.yaml

```diff
@@ -0,0 +1,63 @@
+{{- if .Values.connectors.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "connectors.fullname" . }}
+  labels: {{- include "connectors.labels" . | nindent 4 }}
+  annotations: {{- toYaml .Values.global.annotations | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels: {{- include "connectors.matchLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "connectors.labels" . | nindent 8 }}
+        {{- if .Values.podLabels }}
+        {{- toYaml .Values.podLabels | nindent 8 }}
+        {{- end }}
+        {{- if .Values.podAnnotations }}
+        annotations: {{- toYaml  .Values.podAnnotations | nindent 8 }}
+        {{- end }}
+    spec:
+      imagePullSecrets:
+        {{- include "camundaPlatform.imagePullSecrets" . | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: {{ include "camundaPlatform.image" . | quote }} # has to be camunda/connectors-bundle:0.16.0
```

Comment:
@aabouzaid I don't recall specifying `camundaPlatform.image` as `camunda/connectors-bundle:0.16.0`. Can you please point me out where this happens or how is it mapped?

Reply:
The `camundaPlatform.image` is a [named template](https://helm.sh/docs/chart_template_guide/named_templates/) checks first if the component has an image defined (e.g. `web-modeler.image` which uses a custom image) if it's there it will be used, if not, it will use the global image under `global.image`.

But since now the connectors chart is in the top-level chart, you need to set it like this:

```
{{ include "camundaPlatform.imageByParams" (dict "base" .Values.global "overlay" .Values.connectors) }}
```

later (when we have more top-level components like the connectors) we could simplify it a bit, but it's fine for now.